dropwizard-hibernate/src/test/java/io/dropwizard/hibernate/PersistenceExceptionMapper.java

```diff
@@ -0,0 +1,37 @@
+package io.dropwizard.hibernate;
+
+import org.hibernate.exception.DataException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.persistence.PersistenceException;
+import javax.ws.rs.core.Context;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.ext.ExceptionMapper;
+import javax.ws.rs.ext.Provider;
+import javax.ws.rs.ext.Providers;
+
+@Provider
+public class PersistenceExceptionMapper implements ExceptionMapper<PersistenceException> {
```

Comment:
A pity we can't bundle this class to Dropwizard. The users who will migrate to 5.2 will forced to copy-paste it. 
But we could mention something about it in the migration docs.